src/ArborX_DBSCAN.hpp

```diff
@@ -20,6 +20,7 @@
 #include <ArborX_DetailsKokkosExtAccessibilityTraits.hpp>
 #include <ArborX_DetailsKokkosExtStdAlgorithms.hpp>
 #include <ArborX_DetailsSortUtils.hpp>
+#include <ArborX_DetailsUtils.hpp>
```

Comment:
```suggestion
#include <ArborX_DetailsUtils.hpp> // sortObjects
```
(adjust as necessary)

Reply:
You don't want to add the comment saying what we use?

Reply:
Uggg I was looking at an outdated commit.  Sorry about the noise.